### DISCUSSION TITLE
Track performance of core functions

### DISCUSSION THREAD

**Discussion by corasaurus-hex** [clavascript/clavascript#185] [Ideas]
It would be nice if we could track performance of core functions across releases/commits in order to prevent performance regressions and more easily test performance improvements. I know some languages do this with rather elaborate suites but even something simple to easily compare performance would be great.

I've used https://benchmarkjs.com/ to do testing in Clava locally but I don't know what else is out there.

**Comment by borkdude** [comment]
Moving this to a discussion / idea until we have a concrete plan how to do it.
